integration_tests/pages/confirmCreate.ts

```diff
@@ -11,13 +11,13 @@ export default class ConfirmCreatePage extends Page {
   }
 
   selectYes = (): ConfirmCreatePage => {
+    cy.task('stubPostLicence')
```

Comment:
Moving this here to avoid future potential headaches.
For normal licence creation, stubbing the post method on `clickContinue` was fine. However, if the licence is an early release, `selectYes` _is_ the continue button. This meant that the licence was submitting when it was expecting to be selecting a 'Yes' radio button, and was failing the test due to the POST endpoint not having been mocked yet.
While this isn't currently used, hopefully it will help prevent any issues in future if tests are added explicitly for the early release warning page.